devtools-ci-index.yaml

```diff
@@ -312,3 +310,9 @@
             ci_cmd: '/bin/bash cico_build_deploy.sh'
             svc_name: f8ui
             timeout: '20m'
+        - '{ci_project}-{git_repo}-build-master':
+            git_organization: redhat-developer
+            git_repo: che-starter
+            ci_project: 'devtools'
+            ci_cmd: '/bin/bash cico_build_deploy.sh'
```

Comment:
you are missing `svc_name` to use when running oc deploy as stated at https://github.com/almighty/almighty-jobs/blob/master/devtools-ci-index.yaml#L149